boards/ti/am243x_evm/am243x_evm_am2434_r5f0_0.dts

```diff
@@ -25,6 +25,7 @@
 	aliases {
 		led0 = &ld26;
 		adc0 = &main_adc0;
+		system-timer = &main_timer8;
```

Comment:
The way the system timer is selected currently is by being the 0th enabled timer instance. So as ugly as it would be you could also match this logic by doing `DT_INST(inst, ti_am654_timer)` to find the timer instance used as the system timer without adding this alias.

That said, we need to fix how we select the system timer, and it was probably going to be by adding an alias like this above. So this is fine.